app/controllers/base_drafts_controller.rb

```diff
@@ -102,7 +102,7 @@ def destroy
     Rails.logger.info("Audit Log: Draft #{get_resource.entry_title} was destroyed by #{current_user.urs_uid} in provider: #{current_user.provider_id}")
 
     respond_to do |format|
-      format.html { redirect_to manage_metadata_path, flash: { success: 'Draft was successfully deleted.' } }
+      format.html { redirect_to send("#{plural_resource_name}_path"), flash: { success: 'Draft was successfully deleted.' } }
```

Comment:
Looking through the design comp discussion there have been many requests to specify the type of draft. So rather than saying "draft has been deleted" we would want this to say "variable draft has been deleted"

We're currently using en.yml for this copy within the bar drafts controller and I think it'd be a good thing to continue using going forward.